src/bundle/Controller/LocationController.php

```diff
@@ -41,10 +46,74 @@ public function actionsAction(
                 '_ez_content_view',
                 [
                     'contentId' => $content->id,
-                    'layout' => 'true',
-                    'viewType' => 'full',
                 ]
             )
         );
     }
+
+    public function visibilityAction(
+        Request $request,
+        LocationService $locationService,
+        UiFormFactory $formFactory,
+        TranslatorInterface $translator,
+        NotificationBuilder $notificationBuilder
+    ) {
+        try {
+            $visibilityForm = $formFactory->createLocationsVisibilityForm();
```

Comment:
Does this just handle one location? So maybe it should be renamed `createLocationVisibilityForm`